<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataCollectionDialog</class>
 <widget class="QDialog" name="DataCollectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>174</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="chartWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>1811</width>
     <height>651</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="HRWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>840</y>
     <width>591</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="GSRWidget" native="true">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>840</y>
     <width>591</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="ValenceWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>840</y>
     <width>591</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>174</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="statsButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>174</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Show stats</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>800</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Heart Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>800</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Galvanic Skin Response</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>800</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Emotion valence</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
